profiles/base-town.yaml

```diff
@@ -0,0 +1,38 @@
+---
+Crossing:
+  leather_repair:
+    id: 1544
+    name: randal
+  metal_repair:
+    id: 19093
+    name: catrox
+  bank:
```

Comment:
At least in the Crossing, the bank contains both the deposit room and exchange room. Is this true for Shard?

If so, I recommend calling this `deposit` instead of `bank`.